assets/css/app.css

```diff
@@ -371,6 +371,10 @@ ul.sidebar .sidebar-list .sidebar-sublist a.active {
   float: none !important;
 }
 
+form.bootbox-form input.bootbox-input-checkbox {
```

Comment:
See the codeclimate issue here, you can probably use `.bootbox-form .bootbox-input-checkbox` instead of `form.bootbox-form input.bootbox-input-checkbox`.